From c327de0a238df79e1397373cd87824965d327b32 Mon Sep 17 00:00:00 2001
From: Rong Bao <webmaster@csmantle.top>
Date: Tue, 25 Aug 2026 10:30:04 +0000
Subject: [PATCH] Bug 2065815 - Rename usage of macro Success to X11Success in
 nsXRemoteServer.cpp. r=mossop

This is a fixup to Bug 2065007.

Differential Revision: https://phabricator.services.mozilla.com/D321140
---
 toolkit/components/remote/nsXRemoteServer.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toolkit/components/remote/nsXRemoteServer.cpp b/toolkit/components/remote/nsXRemoteServer.cpp
index 41fa7fa18290d..3e9f149368d14 100644
--- a/toolkit/components/remote/nsXRemoteServer.cpp
+++ b/toolkit/components/remote/nsXRemoteServer.cpp
@@ -110,7 +110,7 @@ bool nsXRemoteServer::HandleNewProperty(XID aWindowId, Display* aDisplay,
 
     // Failed to get property off the window or
     // got a part only
-    if (result != Success || bytes_after != 0) {
+    if (result != X11Success || bytes_after != 0) {
       return false;
     }
 
